### PR TITLE
Proposal: a tutorial for building NFTs on ethereum using IPFS

### PR DESCRIPTION
**Pull request by yusefnapora**
Elevator pitch: People building non-fungible tokens are currently some of the most enthusiastic users of IPFS. Building a tutorial that targets these users will help us improve the developer experience and attract new token builders to use IPFS (and someday Filecoin) as their storage solution.

### CODE REVIEW

File: proposals/ipfs-nft-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+We don't currently offer advice or guidance to users interested in creating NFTs and storing their metadata on IPFS. As a result, we have limited "empathy" for their specific needs, and are largely unaware of their unique pain points, and users building NFTs need to rely on third party resources.
+
+This proposal is to build a guided tutorial that creates an NFT token on ethereum that links to assets stored in IPFS using IPFS cids in the token metadata.
+
+#### Assumptions &amp; hypotheses
+_What must be true for this project to matter?_
+<!--(bullet list)-->
+
+- Users want to store NFT assets on IPFS
```
Comment (momack2) [inline proposals/ipfs-nft-tutorial.md:42]: and Filecoin!

File: proposals/ipfs-nft-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+The benefits of developing this tutorial would be as much for us at Protocol Labs as for our end users. If NFT developers are a key part of our audience / userbase, we should be working to empathize with and understand them as much as possible. As a prerequisite for writing the tutorial, we'll need to understand exactly how NFT devs are currently using IPFS. This will give us insight into things we need to improve and will let us speak more confidently as an organization about the needs of this highly specialized community.
+
+
+#### Confidence
+_How sure are we that this impact would be realized? Label from [this scale](https://medium.com/@nimay/inside-product-introduction-to-feature-priority-using-ice-impact-confidence-ease-and-gist-5180434e5b15)_.
+
+0.1
+<!--Explain why this rating-->
+
+I have no data to support my assertions about impact, just "self conviction" and general hunches.
```
Comment (momack2) [inline proposals/ipfs-nft-tutorial.md:99]: <3 love it

**Comment by rvagg** [comment]
Discussed with @atopal and @zebateira today how we might find a convergence of interest if Sudo takes on #46. If we can map a path to practical storage of NFT assets on Filecoin in that project, then this project can be extended to include that as well.
Scoping this to IPFS for now seems appropriate because there's not a good story yet for doing it on Filecoin, but perhaps we can contribute that piece to the puzzle to make this even more powerful and interesting.

**Comment by mikeal** [comment]
This already happened right? I think we can just merge it 😸

**Pull request merged**